### PR TITLE
fix(dashboard): Crate with fixes

### PR DESCRIPTION
**Pull request by SokratisVidros**
### What changed? Why was the change needed?

See commits

The following screenshots show the UI glitches this PR addresses

Screenshot 2024-12-24 at 12 13 43
Screenshot 2024-12-24 at 12 17 23
Screenshot 2024-12-24 at 13 21 29

### CODE REVIEW

File: apps/dashboard/src/components/create-workflow-button.tsx [hunk 1]
```diff
@@ -73,13 +74,7 @@ export const CreateWorkflowButton = (props: CreateWorkflowButtonProps) => {
...
             <SheetDescription>
               Define the steps to notify subscribers using channels like in-app, email, and more.{' '}
-              <Link
-                target="_blank"
-                to="https://docs.novu.co/concepts/workflows"
-                className="text-foreground-400 inline-flex items-center text-xs underline"
-              >
-                Learn more <RiExternalLinkLine className="inline size-4" />
-              </Link>
+              <ExternalLink href="https://docs.novu.co/concepts/workflows">Learn more</ExternalLink>
```
Comment (scopsy) [inline apps/dashboard/src/components/create-workflow-button.tsx:77]: 👏

File: apps/dashboard/src/components/user-profile.tsx [hunk 2]
```diff
@@ -16,13 +17,15 @@ export function UserProfile() {
...
       }}
     >
-      <UserButton.MenuItems>
-        <UserButton.Action
-          label="Go back to the legacy Dashboard"
-          labelIcon={<RiSignpostFill size="16" color="var(--nv-colors-typography-text-main)" />}
-          onClick={optOut}
-        />
-      </UserButton.MenuItems>
+      {organization.createdAt < new Date('2024-12-24') && (
```
Comment (scopsy) [inline apps/dashboard/src/components/user-profile.tsx]:
```suggestion
     {organization?.createdAt && organization.createdAt < new Date('2024-12-24') && (

```

Comment (scopsy) [reply apps/dashboard/src/components/user-profile.tsx]: Possible null ts error

**Review by scopsy — APPROVED** [review]

File: apps/dashboard/src/components/user-profile.tsx (hunk 2 above)
Comment (SokratisVidros) [reply apps/dashboard/src/components/user-profile.tsx]: Fixed.

File: apps/dashboard/src/context/clerk-provider.tsx [hunk 3]
```diff
@@ -61,15 +36,49 @@ export const ClerkProvider = (props: ClerkProviderProps) => {
...
         elements: {
           formButtonPrimary: cn(buttonVariants({ variant: 'primary' })),
         },
         variables: {
           fontSize: '14px !important',
         },
       }}
-      localization={CLERK_LOCALIZATION}
-      allowedRedirectOrigins={ALLOWED_REDIRECT_ORIGINS}
+      localization={{
```
Comment (SokratisVidros) [inline apps/dashboard/src/context/clerk-provider.tsx:56]: Inlining gives us types in Intellisense ;)

**Pull request merged**